src/modules/Settings/ApiReference/ApiReference.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+type Props = {}
+
+const ApiReference = (props: Props) => {
+  return (
+    <div>ApiReference</div>
+  )
+}
+
+export default ApiReference
```

Comment:
the red minus circle you are seeing here is because there is no newline at the end of your file...
as you code ends at line 11, press enter to have line 12, the the error will go. do it for all the file

Reply:
Done